packages/alpinejs/src/interceptor.js

```diff
@@ -13,6 +13,11 @@ export function initInterceptors(data) {
 
             if (typeof value === 'object' && value !== null && value._x_interceptor) {
                 obj[key] = value.initialize(data, path, key)
+            } else if ((typeof value === 'object' || typeof value === 'function') && value !== null && value._x_accessor) {
+                Object.defineProperty(obj, key, {
```

Comment:
Yeah, that's a good point. I agree it's extra overhead, but it would take reworking all of the interceptors that aren't really meant to be getters/setters.

In a perfect world, there is one unified interface. I'll explore it a little but I want to keep all the existing stuff stable